pom.xml

```diff
@@ -154,6 +154,18 @@
 			<version>3.2.0</version>
 		</dependency>
 
+		<dependency>
+			<groupId>com.koloboke</groupId>
+			<artifactId>koloboke-compile</artifactId>
+			<version>0.5.1</version>
+			<scope>provided</scope>
+		</dependency>
+
+		<dependency>
+			<groupId>com.koloboke</groupId>
```

Comment:
Yes, it's still needed as generated class depends on other classes from impl-common package